### PR TITLE
features(Create models): create users and posts models

### PR DESCRIPTION
**Pull request by GolgothaAksanti**
- create _`.sequelizerc`_ file to direct the path of database folders
- create database _`config.json`_ and _`config.js`_ to configure the local server
- generate users and posts migrations
- create users and posts models
- push my _**create models**_  **Trello** card from **_doing_** to **_review_**

### CODE REVIEW

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require('babel-register');
+const path = require('path');
+
+module.exports = {
```
Comment (alfredruhara) [inline .sequelizerc:4]: Check well the path / based on your folder strcuture

File: Src/App/Controllers/postController.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-console */
+const createPost = (req, res) => {
+  console.log(req.body);
```
Comment (alfredruhara) [inline Src/App/Controllers/postController.js:4]: Avoid pushing console.log code , you should be using process.stdout.write(`req.body`); if you want to push a console.log 

So use console.log only for local developments

File: Src/app.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+/* eslint-disable no-console */
+import express, { json } from 'express';
+import morgan from 'morgan';
+
+// initialization
+const app = express();
+
+// middlewares
+app.use(morgan('dev'));
+app.use(json());
```
Comment (alfredruhara) [inline Src/app.js:11]: Use Body-parse to accept json your app

File: server.js [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
 const app = express();
 
 app.get('/', (req, res) => {
   res.send('Hello world');
 });
 
-app.listen(3000, () => {
-  console.log('Listening on port 3000');
+app.listen(PORT, () => {
+  console.log(`server listening on: http://localhost:${PORT}`);
```
Comment (alfredruhara) [inline server.js:15]: replace that console .log to : process.stdout.write(`server listening on: http://localhost:${PORT});

**Review by alfredruhara — COMMENTED** [review]
Kindly check out the review and improve that in your next PR , This PR Can be merged with the reviews you can fix them in this PR or You can merge the PR and work on them in your next PR .

File: .sequelizerc (hunk 1 above)
Comment (GolgothaAksanti) [reply .sequelizerc:4]: I think the path is well set, isn't it?

**Pull request merged**